### ISSUE TITLE
Slideshow params can't be set remotely

### ISSUE THREAD

**Issue by GoogleCodeExporter** [GloobyBoolga/olpc-xo-picture-frame#1]
```
currently only settable from the command line of "slideshow <pic_delay> 
<reload_delay>"

Ideally, the album properties should contain that information.
One way is to add keywords in the album comments and extract those during album 
download in picasaweb.py.

```

Original issue reported on code.google.com by `abgrall...@gmail.com` on 30 Sep 2012 at 7:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `abgrall...@gmail.com` on 5 Oct 2012 at 9:42
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Done.
Fixed by Revision: e11c4947c70f
```

Original comment by `abgrall...@gmail.com` on 5 Oct 2012 at 9:53
- Changed state: **Fixed**
